tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -97,7 +98,16 @@ func (a *awsProvider) CreateWindowsVM(imageId, instanceType, sshKey string) (rer
 		return fmt.Errorf("failed to get cluster worker IAM, %v", err)
 	}
 
-	instance, err := a.createInstance(imageId, instanceType, sshKey, networkInterface, workerIAM)
+	// PowerShell script to setup WinRM for Ansible
+	userDataWinrm := `<powershell>
+$url = "https://raw.githubusercontent.com/ansible/ansible/devel/examples/scripts/ConfigureRemotingForAnsible.ps1"
```

Comment:
What guarantee do we have that this file will always be present at this location?

Reply:
If we rely on upstream, they may deviate too, it's always better to pin. Since this is a single file, I don't mind keeping a copy in our repo. WDYT @aravindhp?

Reply:
I fine if we keep it in our repo as long as the license allows for it. We also need to ensure that we attribute it correctly.

Reply:
Should I put the file alongside with this PR?

Reply:
Yes as long as you check the license and it allows for it. Please provide proper attribution in the file and your commit / PR message.